src/utils/hooks/useLoadReview.tsx

```diff
@@ -74,16 +79,30 @@ const useLoadReview = ({ reviewId, serialNumber }: UseLoadReviewProps) => {
       })
 
       setReviewSections(sectionsStructure)
-      setLoading(false)
+      setCheckTrigger(true)
     }
-  }, [responsesError, apolloError, data])
+  }, [data])
+
+  useEffect(() => {
+    if (!statusData) return
+    const statuses = statusData?.reviewStatusHistories?.nodes as ReviewStatus[]
+    if (statuses.length > 1) console.log('More than one status resulted for 1 review!')
```

Comment:
Good to check -- are we still getting any situations when this is happening?

Reply:
Yes, that was mainly for dev purpose. I saw that happening when I didn't use the `is_current_status` on the query